controllers/wallet.controller.js

```diff
@@ -172,97 +170,80 @@ exports.userBalance = async (req, res) => {
 }
 
 exports.getGasPrice = async (req, res) => {
```

Comment:
I noticed that there are two functions that have the same name. This could lead to confusion and unexpected behavior, as it's unclear which function would be called at runtime.

To improve readability, I'd suggest giving each function a unique, descriptive name that reflects its purpose or the functionality it provides. 

This way, anyone reading the code can understand what each function does just by looking at its name.